src/types/Tab.ts

```diff
@@ -0,0 +1,3 @@
+export type Tab = {
+  id: string, title: string, content: string,
+};
```

Comment:
```suggestion
export type Tab = {
  id: string,
  title: string,
  content: string,
};
```

It's better to place every prop in the separate line